html-demo/aws/output.tf

```diff
@@ -1,3 +1,3 @@
 output "aws_url" {
-  value = aws_s3_bucket.website_bucket.website_endpoint
+  value = "http://${aws_s3_bucket.website_bucket.website_endpoint}"
```

Comment:
doesn't it offer https?

Reply:
The GCP output for this example was also using http so I just did the same here..